### DISCUSSION TITLE
Can’t Verify Using ”Berlin Schülerausweis II” (Berlin School ID Card)

### DISCUSSION THREAD

**Discussion by necm1** [community/community#61189] [GitHub Education]
### Hi everyone,

Hello!

I can't re-verify my student status using my "Berlin Schülerausweis II", which is a common school ID given by the state / city / school.

Somehow it worked last year & now it's not working.

That's what I get:
![7SmKzag](https://github.com/community/community/assets/8310583/3fe9487b-21a3-4edf-9c6f-741322573b9e)

It mostly tells me:
```
Your image should be one of the following:
Dated school ID
```

My ID is valid until 01.09.2027 which is written on it, but I guess, the progression can't detect / validate German school ID's

**Comment by samus-aran** [comment]
Hi @necm1 👋 

Although it asks for School ID, you can take a picture of multiple pieces of information. If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

We should be able to detect your Student ID but you may try to translate it. We advise you to take a photo of the original next to an English translation. This also include your profile information if you have non english profile details.

Let us know how you get on 👍

**Reply by necm1** [reply]
How should I translate it? Like should I write down everything in english on a blank paper? 😭
